bertopic/_bertopic.py

```diff
@@ -3184,12 +3220,15 @@ def _extract_representative_docs(self,
         Returns:
             repr_docs_mappings: A dictionary from topic to representative documents
             representative_docs: A flat list of representative documents
-            repr_doc_indices: The indices of representative documents
+            repr_doc_indices: Orrdere indices of representative documents
```

Comment:
```suggestion
            repr_doc_indices: Ordered indices of representative documents
```

Reply:
Thanks for the feedback! This is still all a work-in-progress and is likely to change significantly, especially when new features come in, but I'll make sure to change it accordingly if it is still there after a few commits!